### PR TITLE
add cross-env from cross platform ENV setting, fix #5

### LINKED ISSUE #5: add cross-env to support setting PORT in npm script
set ENV in script is not support on windows, so `"PORT=4000 neutrino start"` won't work.

We can use cross-env to set single ENV in any platform. Therefore the workable cross-platform start script would be 

`"start": "cross-env PORT=4000 neutrino start",`

### PR DESCRIPTION
**Pull request by gasolin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,3 +6,9 @@ is extended by all other core neutrino presets.
 
 Modify this preset if there is shared configuration that should be propagated to all other core
 neutrino presets. This preset contains the bulk of the shared Webpack and ESLint configuration.
+
+## Technology
+
+* Webpack
+* babel
```
Comment (eliperelman) [inline README.md]: s/babel/Babel/

File: README.md [hunk 2]
```diff
@@ -6,3 +6,9 @@ is extended by all other core neutrino presets.
...
+* [babel-preset-latest](http://www.2ality.com/2016/09/three-useful-babel-presets.html) enable up-to-date js syntaxs. Will always contain all the “yearly” presets. At the moment, those are: es2015, es2016 and es2017.
```
Comment (eliperelman) [inline README.md]: "enables up-to-date JS syntax"

Comment (eliperelman) [inline README.md]: Replace the smart quotes around "yearly" with plain quotes. Enclose each of the preset names with backticks for highlighting.

File: package.json [hunk 3]
```diff
@@ -14,9 +14,10 @@
     "babel-eslint": "7.0.0",
     "babel-loader": "6.2.5",
     "babel-plugin-istanbul": "2.0.3",
-    "babel-preset-es2015": "6.16.0",
+    "babel-preset-latest": "6.16.0",
```
Comment (eliperelman) [inline package.json]: This will mean a breaking change in this preset and all other presets.

**Comment by eliperelman** [comment]
I've addressed this in #6, but let's split out the cross-env bits from the Babel preset bits.

**Comment by gasolin** [comment]
sure

**Pull request closed without merging**